zc_install/sql/updates/mysql_upgrade_zencart_160.sql

```diff
@@ -88,6 +88,10 @@ UPDATE countries set address_format_id = 5 where countries_iso_code_3 IN ('BEL',
 UPDATE countries set countries_name = 'Åland Islands' where countries_iso_code_3 = 'ALA';
 UPDATE countries set countries_name = 'Réunion' where countries_iso_code_3 = 'REU';
 UPDATE countries set countries_name = "Côte d'Ivoire" where countries_iso_code_3 = 'CIV';
+UPDATE countries set countries_name = 'Bonaire, Sint Eustatius and Saba' 'BQ','BES','1');
```

Comment:
You are missing a WHERE clause on the first UPDATE. Change all of them Bonaire would not be a good idea :)